src/state/store.js

```diff
@@ -263,15 +251,6 @@ export function extractBucketMetadata(transformedData) {
   return etc;
 }
 
-export function isCategorical(probeType, probeKind) {
-  return (
-    (probeType === 'histogram' && probeKind === 'enumerated') ||
-    probeKind === 'categorical' ||
-    probeKind === 'flag' ||
-    probeKind === 'boolean'
-  );
-}
-
 export function fetchDataForGLAM(params) {
```

Comment:
@spasovski @openjck my hope is that y'all can follow along w/ changes to `store.js` and `probe-utils.js`.

Reply:
I see a lot of red so that helps